archlinux/github-repo

```diff
@@ -0,0 +1 @@
+https://github.com/archlinux/archlinux-docker/
```

Comment:
I think this trailing slash is going to lead to some harmless but cosmetically funny-looking links in some generated bits (like `https://github.com/archlinux/archlinux-docker//issues`, for example).

Reply:
No harm in removing it then :)